README.md

```diff
@@ -0,0 +1,176 @@
+
+**Hello Rails React App:** is a hello world project using rails and react.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+ <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">ReactJS</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Random Greetings**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- `Ruby`
+- `Rails`
+- `React`
+- `Redux`
+- `Code editor (VSCode, RubyMine)`
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  git clone https://github.com/musangamfure/hello-rails-react.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  bundle install
+  npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm run build
+  rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec
+```
+
+### Deployment
+
+You can deploy this project using your own deployment solution
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Musangamfura Emmmanuel**
+
+- [GitHub](https://github.com/musangamfure)
+- [Twitter](https://twitter.com/musangamfure)
+- [LinkedIn](https://www.linkedin.com/in/musangamfurae)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Test as rails and react as separate apps**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/musangamfure/hello-rails-react/issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project consider adding a ⭐️
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank [Microverse](https://bit.ly/MicroverseTN)..
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ]  I noticed that your license link appears to be broken or not working as expected. This issue can be frustrating, but there are several potential reasons why it might not be functioning correctly, I can see that you used the wrong path, kindly fix this for your project to look more professional 😉

![Screenshot from 2023-09-19 23-31-06](https://github.com/musangamfure/hello-rails-react/assets/88894525/ab937ab1-e21c-4555-94f1-8efe72042332)